src/main/java/io/reactivex/rxjava3/core/Flowable.java

```diff
@@ -12676,6 +12676,39 @@ public final Completable ignoreElements() {
         return RxJavaPlugins.onAssembly(new FlowableOnBackpressureLatest<>(this));
     }
 
+    /**
+     * Reduces a sequence of two not emitted values via a function into a single value if the downstream is not ready to receive
+     * new items (indicated by a lack of {@link Subscription#request(long)} calls from it) and emits this latest
+     * item when the downstream becomes ready.
+     * <p>
+     * <img width="640" height="245" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/bp.obp.reduce.v3.png" alt="">
```

Comment:
I've created the marble for this operator: `https://raw.githubusercontent.com/wiki/ReactiveX/RxJava/images/rx-operators/Flowable.onBackpressureReduce.png`. Scaled-down size `640 x 354`.

![image](https://raw.githubusercontent.com/wiki/ReactiveX/RxJava/images/rx-operators/Flowable.onBackpressureReduce.png)

Reply:
Thanks!